### ISSUE TITLE
Installation Error (not accessible / not valid URL)

### ISSUE THREAD

**Issue by sutris** [arvatoSCM/dockerize-magento2#14]
Hi,

I want to report the following error when running `bin/console install <host_name>`:
```
ERROR: build path <dev_path>/arvato/magento2-php:latest either does not exist, is not accessible, or is not a valid URL.
```

Thank you!

**Comment by schmengler** [comment]
This has been fixed in Commit 18c3a66948e2b6a61dc09c5ee66ccdc86354e93a (develop branch, not released yet)

**Comment by sutris** [comment]
Thanks @schmengler. I will close this issue now :)
